docs/glossary.rst

```diff
@@ -0,0 +1,55 @@
+Glossary
+========
+
+.. glossary::
+
+    authenticator
+        A software or hardware implementation of the client side of the Web Authentication standard. The browser usually presents the user with multiple options when registering or logging in with a credential. The authenticator handles the cryptographic aspects of the process.
```

Comment:
I find the connection between an authenticator and the browser a bit vague. I think this should be clarified that browser allows you to _choose_which authenticator you want to use_ 

Examples of authenticators: Chrome Password manager, a usb key, iCloud Keychain, a phone